dd-sdk-android-core/src/main/kotlin/com/datadog/android/api/feature/Feature.kt

```diff
@@ -54,6 +54,11 @@ interface Feature {
          */
         const val SESSION_REPLAY_FEATURE_NAME: String = "session-replay"
 
+        /**
+         * Session Replay Resources sub-feature name.
+         */
+        const val SESSION_REPLAY_RESOURCES_FEATURE_NAME: String = "session-replay-resources"
```

Comment:
By adding this here, the feature name for Resources is now defined in two places - in Feature and also in the feature itself. I notice that this is also the case for Session Replay. Do we want to consolidate both of them to use only the constants from Feature and have a SST?